mariner/file_formats/tests/test_cbddlp.py

```diff
@@ -0,0 +1,43 @@
+import hashlib
+import io
+import pathlib
+from unittest import TestCase
+
+import png
+from pyexpect import expect
+
+from mariner.file_formats.ctb import CTBFile
+
+
+class CTBFileTest(TestCase):
+    def test_loading_ctb_file(self) -> None:
+        path = pathlib.Path(__file__).parent.absolute() / "pyramid.cbddlp"
+        ctb_file = CTBFile.read(path)
```

Comment:
same here, let's rename this as `cbddlp_file`. we can leave the class as `CTBFile` for now. I can refactor this a bit later

Reply:
code updated